File: CISC 131 py/133tSpeak.py
def to133t(string):
    text = ""
    for i in range(len(string)):
        if string[i] == "e" or string[i] == "E":
            text += "3"
        elif string[i] == "s" or string[i] == "S":
            text += "5"
        elif string[i] == "o" or string[i] == "O":
            text += "0"
        else:
            text += string[i]
    return text

print(to133t("Hello world!"))

File: CISC 131 c/133tSpeak.c
#include<stdio.h>

void to133t(char _string[]);

int main(){
    char _string[] = "hello"; //{'h', 'e', 'l', 'l', 'o', '\0'};
    to133t(_string);
    printf("hello : %s", _string);
    return 0;
}

void to133t(char _string[]){
    int index = 0;
    while (_string[index] != '\0'){
        if (_string[index] == 'e' || _string[index] == 'E'){
            _string[index] = '3';
        }
        else if (_string[index] == 's' || _string[index] == 'S'){
            _string[index] = '5';
        }
        else if (_string[index] == 'o' || _string[index] == 'O'){
            _string[index] = '0';
        }
        index++;
    }
}
